lib/test/clojure_lsp/feature/move_coll_entry_test.clj

```diff
@@ -10,12 +11,11 @@
 
 (def ^:private uri (h/file-uri "file:///a.cljc"))
 
-(defn zloc-at [pos]
+(defn zloc-at [[row col]]
   (-> @db/db
       (get-in [:documents uri])
       :text
-      (z/of-string {:track-position? true})
-      (z/find-last-by-pos pos)))
+      (parser/loc-at-pos row col)))
```

Comment:
This looks like a small change, but is actually important. By going through the parser, it proves that move-coll-entry works even when starting on whitespace/comment, without needing `:track-position?`. This is the same codepath that `f.refactor/call-refactor` and `f.code-actions/all` follow.